mountpoint-s3/src/main.rs

```diff
@@ -119,6 +120,31 @@ struct CliArgs {
     )]
     pub allow_other: bool,
 
+    #[clap(
+        long,
+        help = "Add the SSE KMS encryption for PUT requests",
+        help_heading = BUCKET_OPTIONS_HEADER,
+        group = "sse_options"
+    )]
+    pub sse_kms: bool,
+
+    // Keeping [sse_kms] and [dsse_kms] in same arg group makes them mutually exclusive arguments
+    #[clap(
+        long,
+        help = "Add the DSSE KMS encryption for PUT requests",
+        help_heading = BUCKET_OPTIONS_HEADER,
+        group = "sse_options"
+    )]
+    pub dsse_kms: bool,
```

Comment:
Drop the comment, or you can instead define the group above the struct like this:

```rust
#[clap(group(
    clap::ArgGroup::new("sse_options")
        .multiple(false)
        .args(&["dsse_kms", "sse_kms"]),
))]
```

Plus update description alike sse_kms.